membership-attribute-service/app/models/Attributes.scala

```diff
@@ -61,7 +64,8 @@ case class Attributes(
     member = isPaidTier || isFriendTier,
     paidMember = isPaidTier,
     recurringContributor = isRecurringContributor,
-    digitalPack = digitalSubscriberHasActivePlan || isPaperSubscriber || isGuardianPatron,
+    supporterPlus = isSupporterPlus,
+    digitalPack = digitalSubscriberHasActivePlan || isPaperSubscriber || isGuardianPatron || isSupporterPlus,
```

Comment:
on second thoughts, how do you indicate if their supporter plus is big enough to get benefits?